packages/suite-web/e2e/run_tests.ts

```diff
@@ -178,8 +181,7 @@ const runTests = async () => {
 
     if (!finalTestFiles.length) {
         console.log('[run_tests.js] nothing to test!');
-
-        return;
+        throw new Error('No test files found.');
```

Comment:
Without this tests will silently pass even if there is error in config and no test files are found.